config/rootfs/debos/scripts/install-firmware.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+FIRMWARE_SITE=https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git
+FIRMWARE_VERSION=d526e044bddaa2c2ad855c7296147e49be0ab03c
```

Comment:
This is fine for now, but it may be useful to be able to override them in the rootfs-configs.yaml as a follow-up improvement.